### DISCUSSION TITLE
Content of HTTP404 page

### DISCUSSION THREAD

**Discussion by hamishwillee** [mdn/mdn-community#184] [Platform] [answered]
We made all links red in #3815. However you can still click the link and get taken to the page that does not exist. The question is "what text should this display".

Ideally we want to tell people that the page does not exist and hint at why. One of missing page that has not been written, missing page that used to exist but was deleted, badly-formed URL (i.e. a typo).

Further it would be good if we could hint at a solution/process for fixing. When I see a broken link I usually start by searching for the link text and/or the URL. This helps me find either badly formed links or good alternatives to wanted links.

@wbamberg @peterbe @chrisdavidmills @peterbe et al

**Comment by hamishwillee** [comment]
A starting point:

> You've followed a link to a page that doesn't exist.
> 
> The page might not have been written yet or may have been deleted. It is also possible that the link URL is incorrect.
> We recommend searching for similar topics on MDN.

**Reply by chrisdavidmills** [reply]
This is a good start. Perhaps we could also include something like "To create or fix this page, please follow the instructions available at our [content repo README.md](#)", and even "This page's source file would exist at [filename.html](#)"?

**Reply by hamishwillee** [reply]
Providing pointers to the instructions is a good idea. Not so sure about pointer to where the source would be. Feels like a bit too much info. So updated proposal:

> You've followed a link to a page that doesn't exist.
> 
> The page might not have been written yet or may have been deleted. It is also possible that the link URL is incorrect.
> We recommend searching for similar topics on MDN.
>
> To create this page, or just fix the link, please follow our instructions on [content repo README.md](https://github.com/mdn/content#contributing-to-the-content-of-mdn-web-docs)

Any comments from anyone else?

**Reply by chrisdavidmills** [reply]
> Any comments from anyone else?

Seems pretty reasonable to me; thanks Hamish!

**Comment by hamishwillee** [comment]
I believe this has converged to a result. Posted in https://github.com/mdn/yari/issues/3931

**Reply by bsmth** [reply, marked as answer]
And I think this has been fixed in https://github.com/mdn/yari/pull/7070

**Comment by bsmth** [comment]
Going to close this as complete, flagging the thread above as the answer. Thanks, all!
